main.go

```diff
@@ -19,26 +21,34 @@ import (
 )
 
 var (
-	VERSION                = "0.0.1"
-	FlagConfigFile         = "config"
-	FlagProvisionerName    = "provisioner-name"
-	EnvProvisionerName     = "PROVISIONER_NAME"
-	DefaultProvisionerName = "rancher.io/local-path"
-	FlagNamespace          = "namespace"
-	EnvNamespace           = "POD_NAMESPACE"
-	DefaultNamespace       = "local-path-storage"
-	FlagHelperImage        = "helper-image"
-	EnvHelperImage         = "HELPER_IMAGE"
-	DefaultHelperImage     = "rancher/library-busybox:1.31.1"
-	FlagServiceAccountName = "service-account-name"
-	DefaultServiceAccount  = "local-path-provisioner-service-account"
-	EnvServiceAccountName  = "SERVICE_ACCOUNT_NAME"
-	FlagKubeconfig         = "kubeconfig"
-	DefaultConfigFileKey   = "config.json"
-	DefaultConfigMapName   = "local-path-config"
-	FlagConfigMapName      = "configmap-name"
-	FlagHelperPodFile      = "helper-pod-file"
-	DefaultHelperPodFile   = "helperPod.yaml"
+	VERSION                   = "0.0.1"
+	FlagConfigFile            = "config"
+	FlagProvisionerName       = "provisioner-name"
+	EnvProvisionerName        = "PROVISIONER_NAME"
+	DefaultProvisionerName    = "rancher.io/local-path"
+	FlagNamespace             = "namespace"
+	EnvNamespace              = "POD_NAMESPACE"
+	DefaultNamespace          = "local-path-storage"
+	FlagHelperImage           = "helper-image"
+	EnvHelperImage            = "HELPER_IMAGE"
+	DefaultHelperImage        = "rancher/library-busybox:1.31.1"
+	FlagServiceAccountName    = "service-account-name"
+	DefaultServiceAccount     = "local-path-provisioner-service-account"
+	EnvServiceAccountName     = "SERVICE_ACCOUNT_NAME"
+	FlagKubeconfig            = "kubeconfig"
+	DefaultConfigFileKey      = "config.json"
+	DefaultConfigMapName      = "local-path-config"
+	FlagConfigMapName         = "configmap-name"
+	EnvConfigMapName          = "CONFIGMAP_NAME"
+	FlagHelperPodFile         = "helper-pod-file"
+	DefaultHelperPodFile      = "helper-pod.yaml"
```

Comment:
This is another breaking change. I could revert that one but the lower case notation is more convenient and if we break sth here already we might as well break this one now.